Tests/Unit/EmogrifierTest.php

```diff
@@ -2121,4 +2118,41 @@ public function emogrifyMovesStyleElementFromHeadToBody()
             $result
         );
     }
+
+    private function htmlStringContainsTagWithAttribute($html, $tagName, $attribute)
+    {
+        return preg_match('/<' . preg_quote($tagName) . '[^>]+' . preg_quote($attribute) . '/', $html) > 0;
```

Comment:
You'll also need to provide `'/'` as the second parameter to `preg_quote` so that the delimiter also will get escaped.